packages/checkbox/src/Checkbox.tsx

```diff
@@ -26,25 +21,52 @@ const Mode = {
 
 type Mode = typeof Mode[keyof typeof Mode];
 
-const wrapperStyleAnimated = css`
-  transition: 300ms opacity ease-in-out;
-`;
+const checkBoxSize = 14;
+const checkAnimationDuration = 100;
+const hypotenusePct = 100 * Math.sqrt(2); // relative distance from corner to corner
+const insetPct = 100 - hypotenusePct;
+const flourishScale = 2.25;
+const flourishTransitionScale = 4;
+const flourishTransitionDelay = -flourishTransitionScale / flourishScale;
```

Comment:
This is pretty cool. Did you think of this yourself or follow a tutorial?

Reply:
I just kinda made it up 😆